src/module/tests/SuccessTest.ts

```diff
@@ -2069,4 +2069,65 @@ export class SuccessTest<T extends SuccessTestData = SuccessTestData> {
         await test.populateDocuments();
         await ActionResultFlow.executeResult(resultAction, test);
     }
+
+    // Optional Rule RG2: TARGET SIZE MODIFIERS
+    protected applyTargetSizeModifiers(){
```

Comment:
Should this be on SuccessTest?

It's the base class for general applicable functionality around testing, but target size modifiers should only be relevant on physical attacks, I think.

I'd prefer that method to be called on the matching attack tests, while its implementation could be shared using a Flow and composition instead of inerhitance.

Also rule relevant sections of this could be moved into a rules file to improve testability and separate the flow from the rules.